### PR TITLE
adding support for Tax Groups API

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

File: connector/fyle_integrations_platform_connector/apis/tax_groups.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                        'attribute_type': 'TAX_GROUP',
+                        'display_name': 'Tax Group',
+                        'value': tax_group['name'],
+                        'source_id': tax_group['id'],
+                        'detail': {
+                            'tax_rate': tax_group['percentage']
+                        }
+                    })
+
+            self.bulk_create_or_update_expense_attributes(tax_attributes, True)
```
Comment (ashwin1111) [inline connector/fyle_integrations_platform_connector/apis/tax_groups.py:29]: Is it possible to update tax group percentage/name in Fyle webapp?

File: connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py [hunk 2]
```diff
@@ -66,11 +67,12 @@ def set_workspace_id(self):
         self.expense_custom_fields.set_workspace_id(self.workspace_id)
         self.corporate_cards.set_workspace_id(self.workspace_id)
         self.reimbursements.set_workspace_id(self.workspace_id)
+        self.tax_groups.set_workspace_id(self.workspace_id)
 
 
     def import_fyle_dimensions(self):
         """Import Fyle Platform dimension."""
-        apis = ['employees', 'categories', 'projects', 'cost_centers', 'expense_custom_fields', 'corporate_cards']
+        apis = ['employees', 'categories', 'projects', 'cost_centers', 'expense_custom_fields', 'corporate_cards', 'tax_groups']
```
Comment (ashwin1111) [inline connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py:75]: we shouldn't do this, we'll end up creating tax groups in intacct and xero where we don't use. Should be doing this optionally.

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: connector/fyle_integrations_platform_connector/apis/tax_groups.py (hunk 1 above)
Comment (NileshPant1999) [reply connector/fyle_integrations_platform_connector/apis/tax_groups.py:29]: yes, it is possible to change @ashwin1111

File: connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py [hunk 3]
```diff
@@ -66,12 +66,15 @@ def set_workspace_id(self):
         self.expense_custom_fields.set_workspace_id(self.workspace_id)
         self.corporate_cards.set_workspace_id(self.workspace_id)
         self.reimbursements.set_workspace_id(self.workspace_id)
+        self.tax_groups.set_workspace_id(self.workspace_id)
 
-
-    def import_fyle_dimensions(self):
+    def import_fyle_dimensions(self, import_taxes):
```
Comment (ashwin1111) [inline connector/fyle_integrations_platform_connector/fyle_integrations_platform_connector.py:71]: This breaks existing integration, can we assign a false value to this?

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**